src/main/Database/File/FileDao.java

```diff
@@ -0,0 +1,31 @@
+package Database.File;
+
+import Database.Dao;
+import File.File;
+import File.FileType;
+import com.google.api.client.util.DateTime;
+import org.bson.types.ObjectId;
+
+import java.io.InputStream;
+import java.util.Optional;
+
+public interface FileDao extends Dao<File> {
+  Optional<File> get(ObjectId id);
+
+  Optional<File> get(String username);
```

Comment:
i think a getAll(String username) function would replace this

Reply:
Replaced in latest commit